dashboard/lib/pd/foorm/workshop_summarizer.rb

```diff
@@ -51,15 +51,14 @@ def self.summarize_answers_by_survey(foorm_submissions, parsed_forms, ws_submiss
             parsed_forms[:general][form_key]
```

Comment:
only change here is switching from indexing on facilitator name to facilitator id